.github/workflows/windows-zip.yml

```diff
@@ -0,0 +1,70 @@
+name: "Build Windows Zip"
+
+# any branch is useful for testing before a PR is submitted
+on: [push, pull_request]
+
+jobs:
+  build:
+    # run on pushes to any branch
+    # run on PRs from external forks
+    if: |
+         (github.event_name != 'pull_request')
+         || (github.event.pull_request.head.repo.id != github.event.pull_request.base.repo.id)
+    name: Building Windows Zip on Linux without plugins
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - uses: actions/setup-node@v2
+      with:
+        node-version: 12
+
+    - name: Install libreoffice
+      run: |
+        sudo add-apt-repository -y ppa:libreoffice/ppa
+        sudo apt update
+        sudo apt install -y --no-install-recommends libreoffice libreoffice-pdfimport
+
+    - name: Install all dependencies and symlink for ep_etherpad-lite
+      run: src/bin/installDeps.sh
+
+    - name: Run the backend tests
+      run: cd src && npm test
+
+    - name: Build the .zip
+      run: src/bin/buildForWindows.sh
+
+    - name: Archive production artifacts
+      uses: actions/upload-artifact@v2
+      with:
+        name: e
+        path: etherpad-lite-win.zip
+
+
+  deploy:
+        # run on pushes to any branch
+    # run on PRs from external forks
+    if: |
+         (github.event_name != 'pull_request')
+         || (github.event.pull_request.head.repo.id != github.event.pull_request.base.repo.id)
+    name: Deploy Windows Zip on Windows
+    needs: build
+    runs-on: windows-latest
+
+    steps:
+    - name: Download all workflow run artifacts
+      uses: actions/download-artifact@v2
+
+    - name: Extract Etherpad
+      run: 7z x e\etherpad-lite-win.zip -oetherpad
+
+    - name: list
+      run: dir etherpad
+
+    - name: Run Etherpad
+      run: |
+            cd etherpad
+            node node_modules\ep_etherpad-lite\node\server.js &
```

Comment:
Ampersand is a POSIX shell script thing, and IIUC the [default shell for Windows is PowerShell Core](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell). Does ampersand also work with PowerShell?

Reply:
[Yup it does.](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_operators?view=powershell-7.1#background-operator-)

Reply:
Is it even powershell?  If so.  Yes.  I just stack overflowed my way through this feeling dirty at every step ..

Reply:
Yes, it is PowerShell Core.